app/controllers/feedbacks_controller.rb

```diff
@@ -5,8 +5,8 @@ class FeedbacksController < ApplicationController
 
 
  get '/feedbacks/new' do
-   if session[:user_type]="student"
-     @student = Student.find(session[:user_id])
+   if authenticate_student!
+     @student = current_user
```

Comment:
You should just be referencing current_user. `current_user` should already be an instance. Using a variable call `@student` starts to make the abstraction difficult to read. Is this a student or the current_user ?